api/api/spec/spec.yaml

```diff
@@ -1016,11 +1185,15 @@ components:
         - $ref: '#/components/schemas/DecoderFile'
       properties:
         name:
-          type: integer
+          type: string
+          description: Decoder name
         position:
           type: string
```

Comment:
This must be an integer

Reply:
Done in c2b9004d9d0f41c13155e75447be7dbc5be00961